### DISCUSSION TITLE
Tooltips in the Configuration wont show up

### DISCUSSION THREAD

**Discussion by wrybit** [jomjol/AI-on-the-edge-device#2521] [General] [answered]
### The Problem

The Tooltip under Configuration will not show up or replace the marker.
Only empty Placeholder strings like eg. _"$TOOLTIP_PostProcessing_NUMBER.DecimalShift"_
And for some Values description with a deeplink into the Manual would be very helpful.

See --> https://www.dropbox.com/s/q4cgs60u2lm2fzz/AIote-Tooltip-bug.png?dl=0

### Version

Release: v15.2.4 (Commit: cbd14a2)

**Comment by dockSquadron** [comment]
same here. compiled on Windows 10 with WSL configured.

**Comment by caco3** [comment]
you did not set it up correctly!
You can not directly use the Repo content but must use the generated zip files!

Please follow the installation instructions!

**Comment by wrybit** [comment, marked as answer]
Thanks for the hint. Replaced the HTML folder. Now it works well.
I followed a youtube tutorial and he did it there like that ;) And I copied that into my setup, too.
My bad. But maybe a hint in the docs or in the ZIP files would prevent this type of user misusage.

Anyway, thanks for your good support!!

**Reply by caco3** [reply]
You could tell the youtuber to correct his wrong instructions.

And it already is documented, see https://jomjol.github.io/AI-on-the-edge-device-docs/Installation/#3-sd-card
